### PR TITLE
Catch exceptions during invocations of updateSemantics from native code

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by jason-simmons** [comment]
@tvolkert

File: shell/platform/android/io/flutter/view/FlutterView.java [hunk 1]
```diff
@@ -681,10 +681,15 @@ private void handlePlatformMessageResponse(int replyId, byte[] reply) {
         }
     }
 
+    // Called by native to update the semantics/accessibility tree.
```
Comment (goderbauer) [inline shell/platform/android/io/flutter/view/FlutterView.java]: What's "native" in this case? I thought the Java stuff is native?

Comment (jason-simmons) [inline shell/platform/android/io/flutter/view/FlutterView.java]: It's called from the engine's C++ code via JNI

Comment (Hixie) [inline shell/platform/android/io/flutter/view/FlutterView.java]: we should just say that. treat "native" as a "bad word", it's too confusing :-)

File: shell/platform/android/io/flutter/view/FlutterView.java [hunk 2]
```diff
@@ -681,10 +681,15 @@ private void handlePlatformMessageResponse(int replyId, byte[] reply) {
...
-        if (mAccessibilityNodeProvider != null) {
-            buffer.order(ByteOrder.LITTLE_ENDIAN);
-            mAccessibilityNodeProvider.updateSemantics(buffer, strings);
+        try {
+            if (mAccessibilityNodeProvider != null) {
+                buffer.order(ByteOrder.LITTLE_ENDIAN);
+                mAccessibilityNodeProvider.updateSemantics(buffer, strings);
+            }
+        } catch (Exception ex) {
+            Log.e(TAG, "Uncaught exception while updating semantics", ex);
```
Comment (Hixie) [inline shell/platform/android/io/flutter/view/FlutterView.java]: Are there exceptions that are ever actually expected? What are the exceptions we're seeing?

**Comment by tvolkert** [comment]
@Hixie see internal bug 62860749

**Pull request merged**